modules/ROOT/pages/contribute.adoc

```diff
@@ -0,0 +1,18 @@
+= Contribute to MuleSoft Documentation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Many of our documentation repositories are open and we welcome your contributions!
+
+To contribute to the English version of this documentation:
+
+. Select *Edit in Github* on the page you wish to modify.
+. Submit a pull request to the git repository.
+. If it is the first time you are submitting a pull request,
+run the https://api-notebook.anypoint.mulesoft.com/notebooks/#d28e8f95e739ee1e1e30d5ec577f4a27[API Notebook].
+If you aren't already logged in to your GitHub account, you can do so using the link provided.
```

Comment:
Should this be the first step or a prereq?

Also, "the link provided" where?

Reply:
Will clarify, thanks!